src/rgw/rgw_rest.cc

```diff
@@ -1093,7 +1087,7 @@ void RGWRESTFlusher::do_start(int ret)
   set_req_state_err(s, ret); /* no going back from here */
   dump_errno(s);
   dump_start(s);
-  end_header(s, op);
+  end_header(s);
```

Comment:
This was by mistake. I double checked every end_header() call that I had modified.